inc/report.php

```diff
@@ -35,3 +36,28 @@ function register_post_type() {
 		]
 	);
 }
+
+/**
+ * Filter the allowed blocks to ensure all our Annual Report blocks are included.
+ *
+ * @param bool|string[]            $allowed_block_types  Array of block type slugs, or boolean to enable/disable all.
+ * @param \WP_Block_Editor_Context $block_editor_context The current block editor context.
+ *
+ * @return bool|string[] Filtered allowed blocks list.
+ */
+function allowed_block_types( $allowed_block_types, $block_editor_context ) {
+
+	// Ensure all our blocks are allowed and not excluded by the theme.
+	$ar_blocks = [
+		'expandable'
```

Comment:
```suggestion
		'wikimedia-annual-report/expandable'
```
Needs to be the full block name

Reply:
Nevermind that's addressed below